oneshot.d/browser-update-default-data

```diff
@@ -0,0 +1,28 @@
+#!/bin/sh
+MOZEMBED_DIR=/home/$USER/.mozilla/mozembed
+BROWSER_DATA_DIR=/usr/share/sailfish-browser/data
+
+if [ ! -d "$MOZEMBED_DIR" ]; then
+    mkdir -p $MOZEMBED_DIR
+fi
+
+if [ ! -f "$MOZEMBED_DIR/ua-update.json" ]; then
```

Comment:
That's simple. I'll enable ```general.useragent.updates.url``` preference  (+interval & retry timeouts) [1][2]. We haven't had yet time to deployment json to the cdn. I've verified the approach with one of server. The UserAgentUpdates.jsm will cache changes to the /home/nemo/.mozilla/mozembed/ua-update.json => that's why we only need to take care of the initial flashing of the device.

Sorry that I cut some corners in the PR description.

[1] Similar to:
https://git.merproject.org/mer-core/gecko-dev/blob/nemo_embedlite/b2g/app/b2g.js#L871

[2]
https://git.merproject.org/mer-core/gecko-dev/blob/nemo_embedlite/netwerk/protocol/http/UserAgentUpdates.jsm#L45